app/controllers/users/new.js

```diff
@@ -22,55 +22,83 @@ export default Ember.Controller.extend({
     this.set("mobilePhone", "");
     this.set("email", "");
     this.set("position", "");
+    this.set("preferredPhone", "");
   },
 
-  formatMobileNumber(){
-    var mobile = this.get('mobilePhone');
-    if(mobile.length){
-      return config.APP.HK_COUNTRY_CODE + this.get('mobilePhone');
+  formatMobileNumber(mobileNumber) {
+    if (mobileNumber.length) {
```

Comment:
I dont understand use case of this function.
We're passing a number and returning the same number in if and else both conditions. Do we even require this function?